examples/00-fluent/mixing_elbow_settings_api.py

```diff
@@ -61,7 +61,10 @@
 # in this example. If you want working units for length to be other than inches
 # (for example, millimeters), make the appropriate change.
 
-solver.tui.define.units("length", "in")
+solver.setup.general.units.set_units(
+    quantity="length",
+    units_name="mm",
+)
```

Comment:
I did not realise that this was in the settings API. Let me comment on the API here, rather than on this change, which is fine.

This is purely a user interface setting. Not only is it irrelevant to a remote user, it's also misleading in a way that could have a negative side effect. After setting the length unit (for instance), the user can then get (or set) a length parameter in the belief that the values are in their chosen unit. Avoidance of this kind of muddled API design is something that has been raised several times. 

Furthermore, the help string is inaccurate: "Set unit conversion factors."  The help   string has to say precisiely what this method does. Additionally, the method name should be updated to reflect that this is purely a Fluent UI setting: e.g., `set_fluent_ui_units`.

@dnwillia-work @h-krishnan

Reply:
Thanks for calling this out, Sean. Since this purely a GUI change and the GUI is not shown in this example, would it be prudent for me to delete this section entirely from all the examples that use it?

Reply:
Yes @ansjsia, great point. If those commands are only relevant for the GUI, I would say they must be removed from all the PyFluent examples, otherwise they will be very misleading to users

Reply:
I've removed all calls to `root.setup.general.units.set_units()` in all the PyFluent examples.